jerry-core/parser/js/js-parser-internal.h

```diff
@@ -273,6 +273,19 @@ typedef struct
 #endif /* PARSER_DUMP_BYTE_CODE */
 } parser_context_t;
 
+/**
+ * @}
+ * @}
+ * @}
+ */
+
+/** \addtogroup mem Memory allocation
+ * @{
+ *
+ * \addtogroup mem_parser Parser memory manager
+ * @{
+ */
+
```

Comment:
I could shrink things a bit (some empty line vanished) but this is the limit we can reach (provided that we follow our own style used for doc comments).
Patch updated.